### DISCUSSION TITLE
Issue in extracting text with regional languages.

### DISCUSSION THREAD

**Discussion by Jisan09** [jsvine/pdfplumber#973] [Ask for help with specific PDFs]
Hi, I have [this PDF](https://docs.primetenders.com/documents/tender/2023/03/24/9626814043637869849/NIT_1.pdf) when i try extract `page.extract_text()` it giving  me some weird texts as output. Is there way to get proper text or  skip saving if the text not proper?

here output sample i got: 
```sLdlornn(hl  Lonillo ornffiflu&  opsd  oJlu.
gp
p. o. o cudr,.1  so I zozsI   pn0h.08,03.2023 euu$puqsfrofl  edlofluq sn6fl  elL6iDL  6lur-rqe6ir  roEOtrb  ono'lg& ot'lqooir  Geofl $q$6lsn6ir6r6i) elgDd6sr6ur  guu$puqoirofl GonTeil
guu$pL+6irdfl&  Gongrb  gg11oroh I gl$dl6Do  oeuiir.r-o  Guoeunonh,  GsnoroJ uoenir.r-orb, afli, guorrf) r-noirr.on&  305A,  orflennnirgfl&dl  sr6lD6D,
u6Dpru dl$pnroeuofl  &'.L@p6q  dlr-rirg, rlonGo@,
Gonrurbq$qjTh ronorr-r-rb  -641 004.```

**Comment by jsvine** [comment]
Hi @Jisan09, the issue here seems to be that this is a scanned PDF, and that the OCR (converting the image to text) has not succeeded well, even before you start working with it in `pdfplumber`. One way to see this is to open the PDF in a standard PDF viewer, select some text, copy it to your clipboard, and then paste it into a text editor.
